assets/stylesheets/components/_toast.scss

```diff
@@ -0,0 +1,44 @@
+#toast-container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  z-index: 6;
```

Comment:
Is this necessary? If so, should this `z-index` belong in `crds-connect` since its very context-specific?